code/game/objects/items/weapons/bee_briefcase.dm

```diff
@@ -67,7 +65,7 @@
 
 		//Release up to 5 bees per use. Without using strange reagent, that means two uses. WITH strange reagent, you can get more if you don't release the last bee
 		for(var/bee = min(5, bees_left), bee > 0, bee--)
-			var/mob/living/simple_animal/hostile/poison/bees/syndi/B = new /mob/living/simple_animal/hostile/poison/bees/syndi(null)
+			var/mob/living/simple_animal/hostile/poison/bees/syndi/B = new /mob/living/simple_animal/hostile/poison/bees/syndi(src)
```

Comment:
does it need this or can it also just be created on the turf of the user?

Reply:
Yeah actually that works fine. 👍